### PR TITLE
feat: add steps tag and create pull req

### PR DESCRIPTION
**Pull request by Jonathan-Casey**
### Changes

- <ONE> add steps tag and create pull req
- <TWO> updates build and publish step

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -94,6 +94,5 @@ jobs:
       - name: build and publish
         run: |
           npm config set //registry.npmjs.org/:_authToken ${{ secrets.NPM_TOKEN }}
-          npm version --no-git-tag-version --yes --exact ${{ steps.timestamp.outputs.stamp }}
-          npx lerna version --no-git-tag-version --yes --exact ${{ steps.timestamp.outputs.stamp }}
-          npx lerna exec -- npm publish --access public --tag=unstable 2>&1
+          npm version patch --force --no-git-tag-version --yes --exact ${{ steps.timestamp.outputs.stamp }}
```
Comment (mttrbrts) [inline .github/workflows/build.yml:97]: Why did you need to change this to `patch`?

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -86,13 +86,31 @@ jobs:
...
         run: |
-          node ./scripts/timestamp.js
+          node ./scripts/tag.js ${{ github.event.release.tag_name }}
 
       - name: build and publish
         run: |
           npm config set //registry.npmjs.org/:_authToken ${{ secrets.NPM_TOKEN }}
-          npm version patch --force --no-git-tag-version --yes --exact ${{ steps.timestamp.outputs.stamp }}
-          npm publish --access public --tag=unstable 2>&1
+          npm version --no-git-tag-version --yes --exact ${{ github.event.release.tag_name  }}
```
Comment (mttrbrts) [inline .github/workflows/npm-publish.yml:97]: Why did we lose the patch here?

File: .github/workflows/build.yml (hunk 1 above)
Comment (Jonathan-Casey) [reply .github/workflows/build.yml]: typo - fixed.
